src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -594,7 +596,7 @@ partial class BoundIncrementOperator : IIncrementExpression
 
     partial class BoundBadExpression
     {
-        protected override OperationKind ExpressionKind => OperationKind.None;
+        protected override OperationKind ExpressionKind => OperationKind.BadExpression;
```

Comment:
"Invalid" is an extremely common term in .NET, for example, InvalidOperationException